tensorrec/tensorrec.py

```diff
@@ -405,6 +405,8 @@ def fit_partial(self, interactions, user_features, item_features, epochs=1, lear
         if self.loss_graph.is_sample_based:
             if (n_sampled_items is None) or (n_sampled_items <= 0):
                 raise ValueError("n_sampled_items must be an integer >0")
+        if (n_sampled_items is not None) and (not self.loss_graph.is_sample_based):
+            logging.warning('n_sampled_items was specified, but the loss graph is not sample-based')
```

Comment:
need to think about the math but we might want to warn if `n_sampled_items == 1 `...degenerate case?

Reply:
I think `n_sampled_items == 1` isn't good practice but isn't necessarily degenerate. If we warn on that, we're saying "1 is too low of a value" but I'm hesitant to define "too low" for all problems. 

For example: Is 2 too low for all problems? Probably... but I don't think we'd warn on that, either.

What do you think?

Reply:
yea, I guess degenerate is to strong. unexpected behavior...but yea I guess we can't really say anything on the "correct" n-sample